packages/client/e2e/limit-checker.spec.ts

```diff
@@ -38,7 +38,9 @@ test.describe("Limit Checker", () => {
       .locator("input")
 
     const lastTradeIdString = await tradeBlotterFirstRowCells.nth(1).innerText()
-    const lastTradeId = isNaN(Number(lastTradeIdString)) ? 0 : Number(lastTradeIdString)
+    const lastTradeId = isNaN(Number(lastTradeIdString))
+      ? 0
+      : Number(lastTradeIdString)
 
     const limitIdString = await limitTableFirstRowCells.nth(1).innerText()
     const limitId = isNaN(Number(limitIdString)) ? -1 : Number(limitIdString)
```

Comment:
aside: why aren't we locating based on the exact ticket name (ccy pair) rather than nth-ing it out of all "EUR" tiles ?